tests/test_fetcher.py

```diff
@@ -143,6 +143,7 @@ def test_30_with_queue(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.text, 'hello')
+        self.assertEqual(self.outqueue.qsize(), 0)
```

Comment:
Maybe not a good place for qsize test, but it's a quick demo to see what's happen if no MultiProcessingQueue.qsize.